docs/learn-DOT.md

```diff
@@ -49,6 +53,59 @@ DOTs will be used to facilitate the consensus mechanism that underpins Polkadot.
 
 DOTs will have the ability to be bonded for a duration of time in order to add a new parachain to the network. The DOTs will be locked during their bonding period and will be released back to the account that bonded them after the duration of the bond has elapsed and the parachain is removed.
 
+### Vesting
+
+DOTs that are allocated to the Polkadot team or awarded to teams as part of 
+Web3 Foundation's grants program will have a vesting schedule attached. Vested
+balances are classified as a separate abstraction to the other locks that are
+placed on your balance. This means that although vested tokens cannot be transferred,
+they can still be used for voting in governance or staked in the staking system.
+
+Vesting schedules are linear release and will unlock a constant amount of tokens
+at every block until all the funds have been made available.
+
+Let's take for example the data structure for the vesting schedule, the `VestingInfo`
+struct:
+
+```rust
+pub struct VestingInfo<Balance, BlockNumber> {
+	/// Locked amount at genesis.
+	pub locked: Balance,
+	/// Amount that gets unlocked every block after `starting_block`.
+	pub per_block: Balance,
+	/// Starting block for unlocking(vesting).
+	pub starting_block: BlockNumber,
+}
+```
+
+There are three fields in the struct above: `locked`, `per_block` and `starting_block`.
+The configuration of these three fields dictate the amount of funds locked, the
+slope of the linear curve for unlocking and the block number for when unlocking will begin.
+
+There are two ways that vesting schedules are created. 
+
+- The first way will be
+part of the genesis configuration of Polkadot, the chain specification will
+read the state of the Polkadot Claims contract on Ethereum for allocations that
+are registered as _vested_ and create the appropriate vesting schedule for them.
+
+- The second way is through a transaction type available in the "Vesting" pallet,
+`vested_transfer`. The vested transfer function allows anyone to create a 
+vesting schedule with a transfer of funds, with the caveat that the account
+for which the vesting schedule will be created does not already have one. To
+call vested transfer successfully you must transfer at least `MinVestedTransfer`
+funds, which is specified as a chain constant.
+
+#### Lazy Vesting
+
+Like [lazy payouts](learn-lazy-payouts), vesting is _lazy_. This means that
+someone must explicitly call either `vest` to vest themselves (the caller) or 
```

Comment:
Should say that calling `vest` just updates the lock to reflect the new (lower) amount that is vesting.